distributed/deploy/ssh.py

```diff
@@ -273,7 +275,8 @@ class SSHCluster(object):
     def __init__(self, scheduler_addr, scheduler_port, worker_addrs, nthreads=0, nprocs=1,
                  ssh_username=None, ssh_port=22, ssh_private_key=None,
                  nohost=False, logdir=None, remote_python=None,
-                 memory_limit=None, worker_port=None, nanny_port=None):
+                 memory_limit=None, worker_port=None, nanny_port=None,
+                 remote_dask_worker=None):
```

Comment:
```suggestion
                 remote_dask_worker='distributed.cli.dask_worker'):
```

Reply:
I modeled remote_dask_worker off remote_python, where the innermost layer sets the default. Is there a reason to do remote_dask_worker differently?

Reply:
When we provide keyword arguments these go into docstrings (which are useful in IDEs and interactive sessions) and into documentation.  

People tend to avoid placing system-specific values into keywords so that they don't get written into docs. 

So for `sys.executable` we avoid it because then we'll have something like `/home/beberg/miniconda/bin/python` in the docs (or whatever the readthedocs build service uses).  However for the constant value of `distributed.cli.dask_worker` it probably does make sense to use.